distributed/scheduler.py

```diff
@@ -6751,17 +6741,14 @@ def update_data(
                 if ts is None:
                     ts: TaskState = parent.new_task(key, None, "memory")
                 ts.state = "memory"
-                ts_nbytes: Py_ssize_t = nbytes.get(key, -1)
+                ts_nbytes = nbytes.get(key, -1)
                 if ts_nbytes >= 0:
                     ts.set_nbytes(ts_nbytes)
-                else:
-                    ts_nbytes = ts.get_nbytes()
+
                 for w in workers:
                     ws: WorkerState = parent._workers_dv[w]
-                    if ts not in ws._has_what:
-                        ws._nbytes += ts_nbytes
-                        ws._has_what[ts] = None
-                        ts._who_has.add(ws)
+                    if ws not in ts._who_has:
```

Comment:
I noticed changes like these in a few different places. You replaced a `ts not in ws._has_what` with a `ws not in ts._who_has`. Assuming the state machine is not corrupt, these are equivalent statements. What motivated this change?

Reply:
pure aesthetics and consistency. Nothing beyond that.